configFile.txt

```diff
@@ -1,16 +1,16 @@
 {
 # DETECTOR
-'geometry'  : 'lime',
+'geometry'  : 'Mango_full',
```

Comment:
can you put back "lime" as default? Or maybe one can make yet another config file for a different detector (like you did for Orange)